doc/website/advanced/custom-iceoryx-platform.md

```diff
@@ -0,0 +1,37 @@
+# Custom iceoryx platforms
+
+The `iceoryx_hoofs/platform` represents the lowest layer in iceoryx. Its task is to establish
+a uniform behavior of low level system calls across all supported platforms. When one would like
+to add support for another platform one can use the cmake switch `-DPLATFORM_PATH` to provide
```

Comment:
I used `set(PLATFORM_PATH "" CACHE PATH "...")` instead of `option(PLATFORM_PATH...`. `set` transforms every user provided path into an absolute path, see: https://cmake.org/cmake/help/latest/command/set.html